### ISSUE TITLE
Finetuning with Undesired Resolution

### ISSUE THREAD

**Issue by hanyangclarence** [THUDM/CogKit#15]
Thanks for your great work!

I would like to lora fine-tune the CogvideoX 1.5 I2V 5B model. But my data resolution is low, like 240 x 240 or 360 x 360. I would like to know whether this would bring negative impact to the fine tuning process. Or would I better change to CogvideoX instead of 1.5 version? Or increase the data resolution?

Thanks for your reply!

**Comment by OleehyO** [comment]
It depends on the `--train_resolution` set in the training script. For example, if it's set to 81x768x1360, then your video must have at least 81 frames (frames exceeding 81 will be downsampled), and the resolution in the video will also be resized to 768, 1360.

**Comment by OleehyO** [comment]
Additionally, we will support multi-resolution training in the coming days

**Comment by hanyangclarence** [comment]
> It depends on the `--train_resolution` set in the training script. For example, if it's set to 81x768x1360, then your video must have at least 81 frames (frames exceeding 81 will be downsampled), and the resolution in the video will also be resized to 768, 1360.

Thanks for the reply.

What I mean is that, if I set the input resolution as, say 17 x 360 x 360, in the training script, and my video data is also 17 frames of 360x360. Will this degrade the model performance during lora fine-tuning?

**Comment by OleehyO** [comment]
It's hard to say; it depends on the quality and quantity of your dataset. After all, for CogVideoX1.5, the number of frames and resolution used in post training are much higher than 17 frames and 360x360. Therefore, excessive fine-tuning under this setting may lead to the loss of some original capabilities. It is recommended that you prepare a validation set and continuously monitor performance changes during training.

**Comment by hanyangclarence** [comment]
okay! Thanks for your advice~
